main.py

```diff
@@ -15,12 +15,9 @@ def build_parser():
 
 
 def find_it(hwnd, parser):
-    _, p = win32process.GetWindowThreadProcessId(hwnd)
-
-    if parser.pid == p:
+    if parser.win_name in win32gui.GetWindowText(hwnd):
```

Comment:
You need to check the name of the EXE file of the running process, not the title of the window